modules/tinymce/src/plugins/wordcount/test/ts/browser/PluginTest.ts

```diff
@@ -85,4 +85,131 @@ describe('browser.tinymce.plugins.wordcount.PluginTest', () => {
     editor.setContent('<p><span>&#8203;&#8203;&#8203;wo&#8203;rd&#8203;&#8203;&#8203;</span></p>');
     await pWaitForWordcount(1);
   });
+
+  it('TINY-8122: Treat $ as a word break', async () => {
+    const editor = hook.editor();
+    await pWaitForWordcount(0);
```

Comment:
You can refactor the test like so:
```suggestion
    const wordCountScenarios = [
      {
        testStr: '<p>$word</p>'
        wordCount: 1
      },
      {
        ...
      }
    ];
    Arr.each(wordCountScenarios, (scenario) => {
      editor.setContent(scenario.testStr);
      await pWaitForWordcount(scenario.wordCount);
    });
```

Reply:
I would prefer to have the `it` be a `context` and each of these scenarios be an `it`, right now if this test fails all you get is `expected x got y` without knowing which of these scenarios failed. But I don't think there's time to fix it for code freeze.